packages/frontend/src/pages/AuthPopupResult.tsx

```diff
@@ -0,0 +1,53 @@
+import { Image, Stack } from '@mantine/core';
+import { IconAlertCircle } from '@tabler/icons-react';
+import { type FC } from 'react';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router-dom';
+import { useMount } from 'react-use';
+import SuboptimalState from '../components/common/SuboptimalState/SuboptimalState';
+import LightdashLogo from '../svgs/lightdash-black.svg';
+
+const AuthPopupResult: FC = () => {
```

Comment:
If the auth was done in a popup, we redirect to this page. 
This page will then communicate back to the main page with the auth status and close itself.